### PR TITLE
doc: add minutes for meeting Aug 14 2017

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

**Comment by mhdawson** [comment]
@nodejs/benchmarking  please review so that I can land/close

**Comment by mhdawson** [comment]
@nodejs/benchmarking please take a look.

File: meetings/2017-08-14/minutes.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  * joining MS benchmarking team and will become more in more active in benchmarking WG.
+
+## Actions from last meeting
+
+* Kunal to do Acme Air runs on community benchmarking machine and report if he sees the
+  same variance of not.
+  * Completed on community, little variance seen, so we should be ok with the 1 run as is done
+    Today - Done
+* Kunal to create PR to add mean (or something else) to Acme air runs.  
+  * Kunal will open an issue in the repo to track, and we.ll remove from this list.
```
Comment (kunalspathak) [inline meetings/2017-08-14/minutes.md]: nit: typo. "and we will remove from this list"

File: meetings/2017-08-14/minutes.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+### Tagged
+* Status of running node core benchmarks [#127](https://github.com/nodejs/benchmarking/issues/127)
+  * good progress made, waiting on some feedback to decided if we are done or not. Once
+    We have initial feedback we can promote further so people know its available.
+* New script to run Node-DC-EIS workload  [#123](https://github.com/nodejs/benchmarking/pull/123)
+  * Meeting will be on Thursday this week.  Kunal to open issue with meeting info so 
+    other people can join if they want.
+  * https://github.com/nodejs/benchmarking/issues/131
+
+* we looked through and discussed/reviewed some of the other more recent active issues as
```
Comment (kunalspathak) [inline meetings/2017-08-14/minutes.md]: nit: "We"

File: meetings/2017-08-14/minutes.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  well.
+
+### General
```
Comment (kunalspathak) [inline meetings/2017-08-14/minutes.md]: not needed tag.

**Comment by mhdawson** [comment]
@kunalspathak thanks for the review.

**Comment by mhdawson** [comment]
Landed as 12ea09993b58895445d407a36076c1b6960ccc64

**Pull request closed without merging**